libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -319,6 +339,12 @@ static void prvRxSelectSet( ss_ctx_t * ctx,
     ctx->rx_callback = ( void ( * )( Socket_t ) )pvOptionValue;
 
     prvIncrementRefCount( ctx );
+
+    ctx->rx_EventGroup = NULL;
```

Comment:
Does the next line make the last unnecessary?

Reply:
Yes, I missed that. Thanks, fixed it.